src/main/java/de/hysky/skyblocker/mixins/RenderFishMixin.java

```diff
@@ -16,6 +18,8 @@ public abstract class RenderFishMixin {
     @ModifyReturnValue(method = "shouldRender", at = @At("RETURN"))
     private boolean skyblocker$render(boolean original, @Local(argsOnly = true) FishingBobberEntity fishingBobberEntity) {
         //if rendered bobber is not the players and option to hide  others is enabled do not render the bobber
-        return Utils.isOnSkyblock() && SkyblockerConfigManager.get().helpers.fishing.hideOtherPlayersRods ? original && fishingBobberEntity.getPlayerOwner() != MinecraftClient.getInstance().player : original;
+        return Utils.isOnSkyblock() && SkyblockerConfigManager.get().helpers.fishing.hideOtherPlayersRods
+				? original && Objects.equals(MinecraftClient.getInstance().player, fishingBobberEntity.getPlayerOwner())
```

Comment:
Don't really know if `Object.equals` instead of `==` is better, not yet familiar with minecraft internals;  but its generally a good practice and `Entity` has well-defined `.equals()`.

Reply:
`Object.equals` checks for `==` first anyway. But there's no real reason for there to be 2 different instances of the player, so the use of `Object.equals` is kinda unnecessary. Doesn't bother me so keep like that unless some other dev says otherwise

Reply:
`Objects#equals` is good practice actually, it's null-safe compared to calling `#equals` on the object itself and handles both referential equality and value equality.